src/views/Recorder/EmptyState.tsx

```diff
@@ -293,6 +296,30 @@ function WarningMessage({
     )
   }
 
+  if (proxyStatus === 'unhealthy') {
+    return (
+      <Callout.Root>
+        <Callout.Icon>
+          <MinusCircle />
+        </Callout.Icon>
+        <Callout.Text>
+          <strong>Proxy is unhealthy</strong>
+          <br />
+          k6 Studio was unable to reach your network. If you have issues
```

Comment:
We use the full name in other parts of the UI, so I'd rename it to `Grafana k6 Studio` for consistency reasons.

Reply:
We check if the proxy can reach quickpizza, so is "was unable to reach your network" factually correct?

Considering we still allow the user to proceed, maybe we should approach this from a different POV: instead of asking the user to try recording and see if they have any issues it tells them to first "verify" that the health check failure is the expected behavior.
Maybe _something_ like this:
```tsx
<>
  Grafana k6 Studio cannot establish connection to the Internet.
  Unless this is expected due to your internal network configuration, check <button>proxy settings</button> or learn more in the <a>troubleshooting guide</a>.
</>
```